### DISCUSSION TITLE
Sequelize v7 for MSSQL with useUTC

### DISCUSSION THREAD

**Discussion by cokiluck** [sequelize/sequelize#17548] [Help & Questions]
Hi,
My project is working fine with sequelize version 6.36.0. I would like to try sequelize version 7. However I don't see the option useUTC when creating Sequelize object in version 7 (there is an option when create Sequelize in version 6: dialectOptions: { options: { useUTC: false}}).
I am using Sequelize with MSSQL server. Could you please give me any solution to set the useUTC:false when creating Sequelize in version 7? Thank you.

**Comment by WikiRik** [comment]
`useUTC` cannot be changed in v7, since we use `DATETIMEOFFSET` as the SQL data type. See https://sequelize.org/docs/v7/models/data-types/#dates

We're looking into other Temporal data types once that reaches stage 4. See #14295
